coreclr.proj

```diff
@@ -0,0 +1,27 @@
+<Project>
+  <Import Project="Directory.Build.props" />
+  <Import Project="Directory.Build.targets" />
```

Comment:
Automatically importing these props and targets sounds evil

Reply:
These imports add the Arcade SDK to this project, which will provide any dummy targets that I missed and hook up any other Arcade things that I missed that we need for adding this to the consolidated Arcade project.